snaps/reference/keyring-api/client-api/objects.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 1
+toc_max_heading_level: 2
+sidebar_label: Objects
+---
+
+# Keyring Client API objects
+
+[Account management Snaps](../../../features/custom-evm-accounts/index.md) use the following objects
+with the [Keyring Client API](index.md).
+
+## `KeyringAccount`
+
+An object representing an account with its properties and capabilities.
+An account object contains:
+
+- `address`: `string` - Account address or next receive address (UTXO).
```

Comment:
I think  these instances might read better as a table, but it's not a critical change... just a suggestion